### PR TITLE
Person meetings attended page

### PR DESCRIPTION
**Pull request by xybrnet**

### CODE REVIEW

File: pombola/south_africa/views/attendance.py [hunk 1]
```diff
@@ -266,3 +267,26 @@ def get_context_data(self, **kwargs):
...
+    def get_context_data(self, **kwargs):
+        context = super(SAPersonAttendanceView, self).get_context_data(**kwargs)
+        # Extract slug and tag provided on url
+        person_slug = self.kwargs['person_slug']
+
+        # Find (or 404) matching objects
+        person = get_object_or_404(Person, slug=person_slug)
+
+        # Get the attendance records from the PMG API, or the cache
+        person_detail = SAPersonDetail(object = person)
```
Comment (xybrnet) [inline pombola/south_africa/views/attendance.py:284]: I'm not sure whether this is the correct way to do this?
Calling methods on the other class to get the data.

The other TemplateViews that do similar pages e.g. appearances: https://github.com/OpenUpSA/pombola/blob/master/pombola/south_africa/views/speechviews.py#L330

don't get data from the API, so no there's no need in reusing some of the functionality in the `SAPersonDetail`.

File: pombola/south_africa/views/person.py [hunk 2]
```diff
@@ -32,6 +32,19 @@ class SAPersonDetail(PersonSpeakerMappingsMixin, PersonDetail):
...
+        # Meanings of attendance field
+
+        #  A:   Absent
+        #  AP:  Absent with Apologies
+        #  DE:  Departed Early
+        #  L:   Arrived Late
+        #  LDE: Arrived Late and Departed Early
+        #  P:   Present
+
+        self.present_values = set(('P', 'DE', 'L', 'LDE'))
```
Comment (xybrnet) [inline pombola/south_africa/views/person.py:46]: And then whether I should set the present values during __init__?

**Review by xybrnet — COMMENTED** [review]
@longhotsummer 
Could you please give me some guidance with the questions I have in these two comments?

File: pombola/south_africa/views/person.py (hunk 2 above)
Comment (longhotsummer) [reply pombola/south_africa/views/person.py:46]: You can also just set them on the class itself, like `important_org_kind_slugs`

File: pombola/south_africa/views/attendance.py (hunk 1 above)
Comment (longhotsummer) [reply pombola/south_africa/views/attendance.py:284]: I think this is fine. It delegates the business logic out of the view and onto SAPersonDetail, which seems reasonable to me.

Comment (xybrnet) [reply pombola/south_africa/views/attendance.py:284]: Great, thank you.

File: pombola/south_africa/views/person.py (hunk 2 above)
Comment (xybrnet) [reply pombola/south_africa/views/person.py:46]: Cool, I'll do that.
Thanks again.

**Pull request closed without merging**